frontend/src/Settings/Indexers/Indexers/Indexer.js

```diff
@@ -113,6 +114,10 @@ class Indexer extends Component {
               </Label>
           }
 
+          <Label kind={kinds.SUCCESS}>
```

Comment:
This shouldn't be green, it's not on/off like the other settings being enabled. `DEFAULT` is probably good here.